test/scripts/add-peers.sh

```diff
@@ -103,9 +103,9 @@ fi
 # Setup RPC node to connect to all sequencer nodes
 if [ "$LAUNCH_RPC_NODE" = "true" ]; then
     echo "🔗 Setting up RPC node to connect to all sequencer nodes..."
-    OP_RPC_TRUSTED_NODES="\"$OP_GETH_SEQ_ENODE\""
+    OP_RPC_TRUSTED_NODES="\"$OP_SEQ_ENODE\""
     if [ "$CONDUCTOR_ENABLED" = "true" ]; then
-        OP_RPC_TRUSTED_NODES="\"$OP_GETH_SEQ_ENODE\",\"$OP_GETH_SEQ2_ENODE\",\"$OP_GETH_SEQ3_ENODE\""
+        OP_RPC_TRUSTED_NODES="\"$OP_SEQ_ENODE\",\"$OP_GETH_SEQ2_ENODE\",\"$OP_GETH_SEQ3_ENODE\""
     fi
     if [ "$RPC_TYPE" = "geth" ]; then
```

Comment:
It not very convenient to use this scripts. Because it will change the config file in the scripts, so it can not be executed more than once. So I recommand we do not use this until we have a better script